packages/fxa-settings/package.json

```diff
@@ -32,7 +32,7 @@
     ]
   },
   "dependencies": {
-    "@apollo/client": "^3.2.2",
+    "@apollo/client": "^3.3.6",
```

Comment:
Apollo docs:
> Prior to Apollo Client 3.3, readQuery would throw MissingFieldError exceptions to report missing fields. Beginning with Apollo Client 3.3, readQuery always returns null to indicate fields were missing.

At some point I was getting the MissingFiedError exception from some tests, but without a good way to find out why or where.  The code and tests have changed a bit since then, so I don't know if this upgrade is necessary, but I don't see any harm in keeping it.